WTWSMS/events/religious_events.txt

```diff
@@ -30,7 +30,15 @@ province_event = {
 	}
 	
 	mean_time_to_happen = {
-		months = 9600
+		months = 8000
+		modifier = {
+			factor = 0.15
+			AND = {
+				NOT = { year = 500 }
+				religion = nicene # In the early parts of this Mod, the religious world was in turmoil with the fall of Christian WRE
```

Comment:
No it doesn't. This was a period of turmoil and massive heresy problems. From the Fall of Rome until 500 AD, there should be a higher chance of heresy spawning.